From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 16:40:35 +0200
Subject: fix_solidlex_destroy_signature

===================================================================
---
 solid/solid/predicate_parser.y | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/solid/solid/predicate_parser.y b/solid/solid/predicate_parser.y
index 04de2e1..583dcb2 100644
--- a/solid/solid/predicate_parser.y
+++ b/solid/solid/predicate_parser.y
@@ -12,7 +12,7 @@ typedef void* yyscan_t;
 void Soliderror(const char *s);
 int Solidlex( YYSTYPE *yylval, yyscan_t scanner );
 int Solidlex_init( yyscan_t *scanner );
-int Solidlex_destroy( yyscan_t *scanner );
+int Solidlex_destroy( yyscan_t scanner );
 void PredicateParse_initLexer( const char *s, yyscan_t scanner );
 void PredicateParse_mainParse( const char *_code );
 
