From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 16:40:35 +0200
Subject: glibc_filesystem

---
 kdecore/io/kfilesystemtype_p.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kdecore/io/kfilesystemtype_p.cpp b/kdecore/io/kfilesystemtype_p.cpp
index 158f51d..3bb1539 100644
--- a/kdecore/io/kfilesystemtype_p.cpp
+++ b/kdecore/io/kfilesystemtype_p.cpp
@@ -56,7 +56,7 @@ KFileSystemType::Type determineFileSystemTypeImpl(const QByteArray& path)
     return kde_typeFromName(buf.f_fstypename);
 }
 
-#elif defined(Q_OS_LINUX) || defined(Q_OS_HURD)
+#elif defined(Q_OS_LINUX) || defined(Q_OS_HURD) || defined(__GLIBC__)
 # include <sys/vfs.h>
 # ifdef QT_LINUXBASE
    // LSB 3.2 has statfs in sys/statfs.h, sys/vfs.h is just an empty dummy header
