From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 16:40:33 +0200
Subject: qt4_designer_plugins_path

===================================================================
---
 kdewidgets/CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/kdewidgets/CMakeLists.txt b/kdewidgets/CMakeLists.txt
index b88b12f..35b0370 100644
--- a/kdewidgets/CMakeLists.txt
+++ b/kdewidgets/CMakeLists.txt
@@ -54,7 +54,7 @@ if(QT_QTDESIGNER_FOUND)
                          )
   endif(NOT WIN32)
 
-  install(TARGETS kdewidgets  DESTINATION ${PLUGIN_INSTALL_DIR}/plugins/designer )
+  install(TARGETS kdewidgets  DESTINATION /usr/lib/${CMAKE_LIBRARY_ARCHITECTURE}/qt4/plugins/designer )
 
   # deprecated widgets
   if(NOT KDE_NO_DEPRECATED)
@@ -83,7 +83,7 @@ if(QT_QTDESIGNER_FOUND)
                            )
     endif(NOT WIN32)
 
-    install(TARGETS kdedeprecated  DESTINATION ${PLUGIN_INSTALL_DIR}/plugins/designer )
+    install(TARGETS kdedeprecated  DESTINATION /usr/lib/${CMAKE_LIBRARY_ARCHITECTURE}/qt4/plugins/designer )
   endif(NOT KDE_NO_DEPRECATED)
 
 
@@ -117,7 +117,7 @@ if(QT_QTDESIGNER_FOUND)
                          )
   endif(NOT WIN32)
 
-  install(TARGETS kdewebkitwidgets  DESTINATION ${PLUGIN_INSTALL_DIR}/plugins/designer )
+  install(TARGETS kdewebkitwidgets  DESTINATION /usr/lib/${CMAKE_LIBRARY_ARCHITECTURE}/qt4/plugins/designer )
 
 
   if (QT_QT3SUPPORT_FOUND)
@@ -152,7 +152,7 @@ if(QT_QTDESIGNER_FOUND)
                              )
      endif(NOT WIN32)
 
-     install(TARGETS kde3supportwidgets  DESTINATION ${PLUGIN_INSTALL_DIR}/plugins/designer )
+     install(TARGETS kde3supportwidgets  DESTINATION /usr/lib/${CMAKE_LIBRARY_ARCHITECTURE}/qt4/plugins/designer )
   endif (QT_QT3SUPPORT_FOUND)
 endif (QT_QTDESIGNER_FOUND)
 
