Index: kde4libs-4.14.17/cmake/modules/KDE4Macros.cmake
===================================================================
--- kde4libs-4.14.17.orig/cmake/modules/KDE4Macros.cmake
+++ kde4libs-4.14.17/cmake/modules/KDE4Macros.cmake
@@ -1011,6 +1011,11 @@ endmacro(_KDE4_ADD_MANIFEST)
 
 macro (KDE4_ADD_EXECUTABLE _target_NAME)
 
+   if (POLICY CMP0064)
+      cmake_policy(PUSH)
+      cmake_policy(SET CMP0064 OLD)
+   endif()
+
    kde4_check_executable_params( _SRCS _nogui _test ${ARGN})
 
    set(_add_executable_param)
@@ -1063,6 +1068,10 @@ macro (KDE4_ADD_EXECUTABLE _target_NAME)
       target_link_libraries(${_target_NAME} ${QT_QTMAIN_LIBRARY})
    endif (WIN32)
 
+   if(POLICY CMP0064)
+      cmake_policy(POP)
+   endif()
+
 endmacro (KDE4_ADD_EXECUTABLE)
 
 
