From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 16:40:35 +0200
Subject: set_cmake_policies

===================================================================
---
 cmake/modules/KDE4Macros.cmake | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/cmake/modules/KDE4Macros.cmake b/cmake/modules/KDE4Macros.cmake
index 6a75f32..39a2804 100644
--- a/cmake/modules/KDE4Macros.cmake
+++ b/cmake/modules/KDE4Macros.cmake
@@ -1011,6 +1011,11 @@ endmacro(_KDE4_ADD_MANIFEST)
 
 macro (KDE4_ADD_EXECUTABLE _target_NAME)
 
+   if (POLICY CMP0064)
+      cmake_policy(PUSH)
+      cmake_policy(SET CMP0064 OLD)
+   endif()
+
    kde4_check_executable_params( _SRCS _nogui _test ${ARGN})
 
    set(_add_executable_param)
@@ -1063,6 +1068,10 @@ macro (KDE4_ADD_EXECUTABLE _target_NAME)
       target_link_libraries(${_target_NAME} ${QT_QTMAIN_LIBRARY})
    endif (WIN32)
 
+   if(POLICY CMP0064)
+      cmake_policy(POP)
+   endif()
+
 endmacro (KDE4_ADD_EXECUTABLE)
 
 
