From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Tue, 12 Jul 2016 21:27:33 +0200
Subject: test_image-x-compressed-xcf

---
 kdeui/tests/kiconloader_unittest.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kdeui/tests/kiconloader_unittest.cpp b/kdeui/tests/kiconloader_unittest.cpp
index b3d04e6..672d867 100644
--- a/kdeui/tests/kiconloader_unittest.cpp
+++ b/kdeui/tests/kiconloader_unittest.cpp
@@ -216,7 +216,7 @@ private Q_SLOTS:
         QTest::newRow("x-office-document icon") << "x-office-document" << "x-office-document.png";
         QTest::newRow("unavailable generic icon") << "application/x-font-vfont" << "application-octet-stream.png";
         QTest::newRow("#184852") << "audio/x-tuxguitar" << "audio-x-generic.png";
-        QTest::newRow("#178847") << "image/x-compressed-xcf" << "image-x-generic.png";
+        QTest::newRow("#178847") << "image/x-compressed-xcf" << "image-x-compressed-xcf.png";
 
         if (sharedMimeInfoVersion >= KDE_MAKE_VERSION(0, 40, 0)) {
             QTest::newRow("mimetype generic icon") << "application-x-fluid" << "x-office-document.png";
