From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Tue, 12 Jul 2016 21:27:32 +0200
Subject: use_an_oxygen5_file

---
 kdecore/tests/kmimetypetest.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kdecore/tests/kmimetypetest.cpp b/kdecore/tests/kmimetypetest.cpp
index 6f155b2..ddf2973 100644
--- a/kdecore/tests/kmimetypetest.cpp
+++ b/kdecore/tests/kmimetypetest.cpp
@@ -252,7 +252,7 @@ void KMimeTypeTest::testFindByPathUsingFileName_data()
     if (KStandardDirs::locate("icon", "oxygen/").isEmpty()) {
         kWarning() << "oxygen not found";
     } else {
-        QString fh = KStandardDirs::locate( "icon", "oxygen/22x22/places/folder.png" );
+        QString fh = KStandardDirs::locate( "icon", "oxygen/base/22x22/places/folder.png" );
         QVERIFY( !fh.isEmpty() ); // if the file doesn't exist, please fix the above to point to an existing icon
         QTest::newRow("png image") << fh << "image/png";
     }
